src/renderers/shared/fiber/ReactFiberErrorLogger.js

```diff
@@ -29,11 +29,23 @@ function logCapturedError(capturedError: CapturedError): void {
     return;
   }
 
+  // Duck-typing
+  const error = (capturedError.error: any);
+  if (
+    error == null ||
+    typeof error.message !== 'string' ||
+    typeof error.name !== 'string' ||
+    typeof error.stack !== 'string'
+  ) {
+    // Something other than an error was thrown.
+    // TODO: What is the correct behavior here?
```

Comment:
Seems like we should still report it? I'd imagine we branch `message, name, stack` initial assignment on whether `error` exists or not, and make `message` be `error` coerced to string if it's missing.